aioxmpp/service.py

```diff
@@ -1358,20 +1414,32 @@ def decorator(f):
     return decorator
 
 
-def is_iq_handler(type_, payload_cls, coro):
+def is_iq_handler(type_, payload_cls, coro, kwargs=None):
```

Comment:
``is_iq_handler`` should have the possible kwargs explicit, like:

```python
def is_iq_handler(type_, payload_cls, core, *, with_send_reply=False):
```

and always check them completely. It is used for unit-testing purposes, and since the argument changes the interface, it needs to be checked in all cases.